OutOfSchool/OutOfSchool.WebApi/Services/SocialGroupService.cs

```diff
@@ -96,11 +95,18 @@ public async Task<SocialGroupDto> Update(SocialGroupDto dto, LocalizationType lo
     {
```

Comment:
Seems we have to mark that `dto` with `[NotNull]` attribute and use `ArgumentNullException.ThrowIfNull` because  this method should not accept nulls there